collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/AutoSendTest.kt

```diff
@@ -66,8 +66,7 @@ class AutoSendTest {
 
         testDependencies.scheduler.runDeferredTasks()
 
-        mainMenuPage
-            .clickViewSentForm(1)
+        mainMenuPage.clickViewSentForm(1)
```

Comment:
What's the point of checking this here if we do not check the status (success/failed)? No matter if sending failed or not the form will be there.

Reply:
A bunch of other tests fail if I mess with code that updates instance statuses after sending so I think this is ok for now.